app/assets/javascripts/shared/sessions.js

```diff
@@ -0,0 +1,31 @@
+document.addEventListener('turbolinks:load', function() {
+  if ($('body.unauthenticated').length) {
+    var charCount = 0,
+        strings = [
+          'Tame the output from multiple scan tools, email threads, and notes',
+          'Deliver quality security assessments at light speed',
+          'Focus less on tedious reporting tasks and more on testing',
+          'Ditch the overhead that drags down project delivery'
+        ],
+        randomStringIndex = Math.floor(Math.random() * strings.length),
+        randomString = strings[randomStringIndex];
+
+    setTimeout(function() {
+      setInterval(function() {
+        if (charCount <= randomString.length) {
+          $('[data-behavior~=mission-text]').text(randomString.substring(0, charCount));
+          charCount += 1;
+        }
+      }, 20);
+    }, 500);
+
+    $('[data-behavior~=logo-icons]').addClass('expand');
```

Comment:
Since this is adding the `expand` class on page load, is there any difference adding `expand` to the markup directly https://github.com/dradis/dradis-ce/pull/1004/files#diff-f43fe075643e681b2c01c2f853bb0c4299d135b47fcbd4da96890d521c49e3ebR22?

Reply:
If we add the class directly to the element then it will instantly show the icons without doing the animation.